tests/integration/site-details/same-activity-dates/same-activity-dates.test.js

```diff
@@ -0,0 +1,220 @@
+import { JSDOM } from 'jsdom'
+import { getByRole, getByText } from '@testing-library/dom'
+import { createServer } from '~/src/server/index.js'
+import { statusCodes } from '~/src/server/common/constants/status-codes.js'
+import {
+  getExemptionCache,
+  updateExemptionMultipleSiteDetails
+} from '~/src/server/common/helpers/session-cache/utils.js'
+
+jest.mock('~/src/server/common/helpers/session-cache/utils.js')
+
+describe('Same activity dates page', () => {
+  let server
+
+  const mockExemption = {
+    id: 'test-exemption-123',
+    projectName: 'Test Project',
+    multipleSiteDetails: {
+      multipleSitesEnabled: true
+    }
+  }
+
+  jest.mocked(getExemptionCache).mockReturnValue(mockExemption)
+  jest.mocked(updateExemptionMultipleSiteDetails).mockReturnValue({})
+
+  beforeAll(async () => {
+    server = await createServer()
+    await server.initialize()
+  })
+
+  afterAll(async () => {
+    await server.stop()
+  })
+
+  test('should display the same activity dates page with correct content', async () => {
+    const { result, statusCode } = await server.inject({
+      method: 'GET',
+      url: '/exemption/same-activity-dates'
+    })
+
+    expect(statusCode).toBe(statusCodes.ok)
+
+    const { document } = new JSDOM(result).window
+
+    expect(
+      getByRole(document, 'heading', {
+        level: 1,
+        name: 'Are the activity dates the same for every site?'
+      })
+    ).toBeInTheDocument()
+    expect(getByText(document, mockExemption.projectName)).toBeInTheDocument()
+
+    const yesRadio = getByRole(document, 'radio', {
+      name: 'Yes, the dates are the same for every site'
+    })
+    expect(yesRadio).toHaveAttribute('type', 'radio')
+
+    const noRadio = getByRole(document, 'radio', {
+      name: 'No, at least one site has different dates'
+    })
+    expect(noRadio).toHaveAttribute('type', 'radio')
+
+    expect(
+      getByText(document, 'Yes, the dates are the same for every site')
```

Comment:
`getByLabelText` will give us a11y checks for free, confirming that an input is associated with the label